### ISSUE TITLE
Migrate documentation to GitHub Pages

### ISSUE THREAD

**Issue by rprouse** [nunit/nunit.org#26]
We have discussed this several times in the mailing list, but haven't made any progress, so I wanted to capture the discussion here. We can change the title of the issue once we make a final decision on how we want to proceed.

The options we have talked about are;

- Move our documentation and possibly the entire NUnit site to use GitHub pages
- Host the documentation in one of the GitHub repository wikis
- Others?

I read a blog post this morning about how GitHub uses GitHub pages to provide all the documentation for GitHub. Their documents are also versioned which would be useful for us. The blog post might give us some direction, https://github.com/blog/1939-how-github-uses-github-to-document-github

I'm not sure if this issue belongs here or in the main repository where we can track it with the release milestones.

**Comment by CharliePoole** [comment]
Actually, it is in the nunit repo as nunit/nunit#168. We also had nunit/nunit#167 for the "interim documentation." That was closed once we set up nunit.org.

I haven't focused heavily on this, since I had scheduled it after the betas and nobody had questioned that. I'll copy some of the discussion we had on the mail list into the issue.

On the points you mention, IIRC, my last proposal was to use the wiki for documentation, github pages for the website and straight html for the "legacy site" which would become a subdomain. There was a bit of feedback, but not much, so I figured on going ahead that way.

I think there's a small amount of design work needed before we start writing or converting docs: the structure of the new docs and some guidelines on style and content.

I'll expand on the above in the nunit issue.
